src/System.Text.Json/src/System/Text/Json/Serialization/Converters/JsonValueConverterByte.cs

```diff
@@ -27,9 +27,9 @@ public override void Write(byte value, Utf8JsonWriter writer)
             writer.WriteNumberValue(value);
         }
 
-        public override void Write(Span<byte> escapedPropertyName, byte value, Utf8JsonWriter writer)
+        public override void Write(in JsonEncodedText propertyName, byte value, Utf8JsonWriter writer)
```

Comment:
Why are these being passed with `in`?

Reply:
My guess is that Steve wants to make sure that the struct (`JsonEncodedText`) does not get copied when it's passed to `Write`. afaik it's not guaranteed otherwise